src/pages/MFARestoreScreen.vue

```diff
@@ -49,7 +53,18 @@ onBeforeMount(async () => {
     show: true,
     message: 'Loading metadata...',
   }
-  dkgShare = JSON.parse(storage.local.getItem('pk') as string)
+  const localSession = storage.session.getItem('userInfo')
+  if (localSession) {
+    userInfoSession = JSON.parse(localSession)
+  }
+  if (userInfoSession?.loginType === 'firebase') {
```

Comment:
Same, let's have a list of bearer type (non-social) authentication methods and use a set membership check.